generator.yaml

```diff
@@ -157,4 +179,4 @@ ignore:
       - Workforce
       - Workteam
     operations:
```

Comment:
do you mind adding the following as part of your PR to remove tags from the all resources including this one?

```
ignore:
...
    shape_names:
      - TagList
```